### DISCUSSION TITLE
Time Slider for Heatmap

### DISCUSSION THREAD

**Discussion by MarcelKuhn** [opengeos/leafmap#501] [Q&A] [answered]
Hi together,
first of all, I wanted to say, that leafmap is very impressive, very good work :-)

I have just a short question: Is it possible to add the time slider function to a heatmap? 
Something like this:

`m.add_heatmap(
        df,
        latitude="latitude",
        longitude="longitude",
        value="pop_max",
        name="Heat map",
        radius=15,
    )
    m.add_time_slider(date="time_col")`

If not, it would be a great feature to explore distrubtion values over time on a map...

Thanks a lot
Marcel

**Comment by giswqs** [comment, marked as answer]
Timeslider only supports raster data layersz
